### PR TITLE
fix: Simplify preview command (LPS-152388)

### PR DESCRIPTION
**Pull request by p2kmgcl**
Basically I have rewritten the whole preview script (it was a bit hard to follow). Now it does less things, and parameters should not be updated on reload.

### CODE REVIEW

**Comment by victorg1991** [comment]
Está muy bien eh?

**Comment by victorg1991** [comment]
Approved 👍

**Comment by sandrodw3** [comment]
La veo genial, por mí para alante 😄

**Comment by p2kmgcl** [comment]
Release to test this feature: [2.0.2-rc1](https://www.npmjs.com/package/generator-liferay-fragments/v/2.0.2-rc1)

File: src/preview/assets/preview.js [hunk 1]
```diff
@@ -1,192 +1,118 @@
...
-    renderSelect(
-      pageTemplateSelect,
-      projectContent.pageTemplates.map((pageTemplate) => ({
-        value: pageTemplate.slug,
-        label: pageTemplate.metadata.name,
-      }))
-    );
+    form.elements.collection.value = url.searchParams.get('fieldsetId');
+    form.elements.collection.value = url.searchParams.get('pageTemplate');
+    form.elements.collection.value = url.searchParams.get('collection');
```
Comment (victorg1991) [inline src/preview/assets/preview.js]: This is weird, is correct?

**Comment by victorg1991** [comment]
In general it works fine 👏 
Just a couple of thing I noticed:

* if you open the url without query string like localhost:xxx/fragment-preview the fragment type is selected but you cannot choose any, you have to switch to page templates and back to fragments.
* When changing from fragments to page template and the user doesn't have page template the fragment is kept rendered

File: src/preview/assets/preview.js (hunk 1 above)
Comment (p2kmgcl) [reply src/preview/assets/preview.js]: :facepalm: no it isn't

**Comment by p2kmgcl** [comment]
Fixed :smile:

File: src/preview/assets/preview.js [hunk 2]
```diff
@@ -1,192 +1,136 @@
...
+    fieldsetId === 'pageTemplatesPreview' &&
+    form.elements.pageTemplate.value
+  ) {
+    const pageTemplateSlug = form.elements.pageTemplate.value;
+    preview.src = `/fragment-preview?pageTemplate=${pageTemplateSlug}`;
   } else {
-    fragmentsPreview.className = '';
-    pageTemplatesPreview.className = 'hide';
+    preview.src = '/fragment-preview';
+  }
```
Comment (p2kmgcl) [inline src/preview/assets/preview.js:99]: I updated this part checking the fieldsetId value

**Review by victorg1991 — APPROVED** [review]
LGTM :)
